nesta/core/schemas/tier_1/remappings/health_scanner.json

```diff
@@ -0,0 +1,167 @@
+{
+    "type_of_entity": {
+        "crunchbase": "company",
+        "meetup": "meetup",
+        "nih": "project"
+    },
+    "id": {
+        "meetup": "id_of_group",
+        "nih": "id_of_project"
+    },
+    "name": {
+        "crunchbase": "name_parent_organisation",
+        "meetup": "name_of_group",
+        "nih": "name_of_project"
+    },
+    "description": {
+        "crunchbase": "textBody_descriptive_organisation",
+        "meetup": "textBody_descriptive_group",
+        "nih": "textBody_descriptive_project"
+    },
+    "brief": {
+        "crunchbase": "textBody_summary_organisation",
+        "meetup": "textBody_abstract_project"
+    },
+    "title": {
+        "nih": "title_of_project"
+    },
+    "start_date": {
```

Comment:
I changed my mind, I think we should use `date_*` for time related keys:
- `date_start`
- `date_end`
- `date_update`